jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -641,9 +641,9 @@ def apply_RGB_presets(self):
             default_opacity = 1 / math.log2(n_visible)
         # Sample along a colormap if we have too many layers
         if n_visible == 2:
-            preset_colors = [preset_colors[0], preset_colors[3]]
+            preset_colors = [preset_colors[1], preset_colors[4]]
         elif n_visible == 3:
-            preset_colors = [preset_colors[0], preset_colors[2], preset_colors[3]]
+            preset_colors = [preset_colors[1], preset_colors[2], preset_colors[4]]

```

Comment:
this surprised me at first (I expected just a change to `preset_colors`), but I guess this is just to avoid the purple on the end and the purple switched sides?

Reply:
Exactly.